File: resolucoes/python/lista01/ex020.py
# Desenvolva um programa que leia um número inteiro e mostre se ele é PAR ou ÍMPAR.

num = int(input('Digite um número inteiro: '))

if num % 2 == 0:
    print(f'O número {num} é par')
else:
    print(f'O número {num} é ímpar')

File: resolucoes/JavaScript/lista01/ex020.js
const prompt = require('prompt-sync')();

function main() {

let numero = Number(prompt("Digite um número inteiro:"))

if (numero % 2 == 0) {
    console.log("O número é PAR.")
} else {
    console.log("O número é ÍMPAR.")
}
}

module.exports = {
    main
}